BUILD

```diff
@@ -30,6 +45,13 @@ java_library(
     ],
 )
 
+java_library(
+    name = "telemetry_tools",
```

Comment:
somewhat unrelated - we should pick a convention between hyphens and underscores for target names. Below you have opentelemetry-javaagent for comparison

Reply:
agree